gradle.properties

```
@@ -1,21 +1,22 @@
 org.gradle.jvmargs=-Xmx2G
 
 # Check versions on https://modmuss50.me/fabric.html
-minecraft_version=1.20.2
-yarn_mappings=1.20.2+build.2
-enabled_platforms=fabric,forge
+minecraft_version=1.20.4
+yarn_mappings=1.20.4+build.3
+enabled_platforms=fabric
```

Maybe add a comment? 
```suggestion
enabled_platforms=fabric # Forge disabled for now
```